### ISSUE TITLE
Effort vs accomplishment

### ISSUE THREAD

**Issue by molliem** [gender-bias/gender-bias#8]
Letters for women are more likely to highlight effort (she is hard-working) instead of highlighting accomplishments (her research is groundbreaking).

Goal: Develop code that can read text for effort statements. If the text includes more effort statements, than accomplishment statements; return a summary that directs the author to add statements related to accomplishment.

**Comment by cmd16** [comment]
Started work on this issue by creating two wordlists and a driver program to count words that fall in either category. [](https://github.com/molliem/gender-bias/pull/13)

**Comment by j6k4m8** [comment]
_Grindstone_ adjectives (Trix & Psenka, 2003):
> ‘hardworking’, ‘conscientious’, ‘dependable’, ‘meticulous’, ‘thorough’, ‘diligent’, ‘dedicated’, and ‘careful’.

Occur in 34% letters for women, vs 23% letters for men

**Comment by molliem** [comment]
Several of these words are included in our female-coded word list.  I'll double check it and add in additional words.  Several articles highlight the fact that one of the big differences between letters form women vs. men is that letters for women describe effort or luck, while letters for men link success to intelligence and leadership.

**Comment by ProfJanetDavis** [comment]
Since this issue is still open, I'll add my comments here. @molliem or @j6k4m8 , let me know if you'd like me to make a separate issue instead.

The existing detector flags grindstone words even if I am not describing the candidate, but my own actions or beliefs as the writer. It seems like this should be possible to address with a little more sophisticated NLP.

The sentences in this screenshot are taken from an actual letter I wrote recently.
![Screen Shot 2020-07-02 at 3 11 05 PM](https://user-images.githubusercontent.com/19829843/86413661-a6462880-bc76-11ea-9761-5844646c16ba.png)

**Comment by j6k4m8** [comment]
Yes we definitely should be leaning on some heftier NLP to solve this. (I want to replace ALL of our detectors that are simple word-detectors, for that matter.) The above is an excellent example. 

I'd love to start collecting a list of test-documents (perhaps contrived, single-sentence examples like these?) that we can use in order to expect certain results. Feelings on that?

I'll try out a fix for this in an upcoming PR unless you'd like a student to tackle it?

**Comment by ProfJanetDavis** [comment]
Thanks for asking, @j6k4m8. I think we'll be entirely focused on the web interface for the next week, if not two.

@molliem and I spoke about this issue before I wrote about it here. She encouraged me to post a screenshot with a minimal example. I wonder whether these simple examples should go in complete example letters, or in pending test cases.
